### PR TITLE
Remove Edit Contention Button on Unidentified issues

### PR DESCRIPTION
**Pull request by Sjones352**
Connects #11451

### Description
 The edit-contention-text button is enabled for unidentified issues which should not be true because user edits the title on Intake creation. Upon further research, while this feature is enabled unidentified issues, it does not allow you to save the edited text. I tested it both in local and UAT and this button is available. However it is not available in prod because we had an issue this morning with user wanting to edit the context text on an unidentified issues.

Screen Shot 2019-07-18 at 2 32 50 PM

**Removed**
Screen Shot 2019-07-18 at 2 35 29 PM

### CODE REVIEW

File: client/app/intake/pages/addIssues.jsx [hunk 1]
```diff
@@ -154,7 +154,8 @@ export class AddIssuesPage extends React.Component {
         <div>
           { requestIssues.map((issue) => {
             const editableContentionText = Boolean(
-              formType !== FORM_TYPES.APPEAL.key && !issue.category && !issue.ineligibleReason);
+              formType !== FORM_TYPES.APPEAL.key &&
+              !issue.category && !issue.ineligibleReason && !issue.isUnidentified);
```
Comment (pkarman) [inline client/app/intake/pages/addIssues.jsx]: looks good -- do we need a test?

Comment (Sjones352) [reply client/app/intake/pages/addIssues.jsx]: Makes sense. I will write one.

Comment (Sjones352) [reply client/app/intake/pages/addIssues.jsx]: added a test

File: spec/feature/intake/higher_level_review/edit_spec.rb [hunk 2]
```diff
@@ -1493,6 +1493,19 @@ def click_cancel(visit_page)
...
+
+      context "when review has unidentified issues and non-ratings" do
+        let!(:issue) do
+          create(:request_issue, :unidentified, decision_review: higher_level_review,
+                                                contested_issue_description: "This is unidentified")
+        end
+
+        scenario "do not show edit contention text on unidentified issues" do
+          visit "higher_level_reviews/#{higher_level_review.uuid}/edit"
+          expect(RequestIssue.where(contested_issue_description: "This is unidentified")).to_not be_nil
```
Comment (pkarman) [inline spec/feature/intake/higher_level_review/edit_spec.rb]: this db call is unnecessary here I think. we just want to make sure the text is on the page

```ruby
expect(page).to have_content("This is unidentified")
```

**Pull request merged**